db/consistency_level.cc

```diff
@@ -23,8 +24,64 @@
 #include "locator/abstract_replication_strategy.hh"
 #include "locator/network_topology_strategy.hh"
 #include "heat_load_balance.hh"
+#include "tools/build_info.hh"
 
 namespace db {
+namespace {
+
+// Check the effective replication map consistency:
+// we have an inconsistent effective replication map in case we have a data center
+// with the number of replica nodes higher than it's replication factor.
+void validate_replica_dc_replication_factor(const locator::effective_replication_map& erm,
+                                            const inet_address_vector_replica_set& replicas,
+                                            const consistency_level cl) {
+    // Skip for release builds.
+    if constexpr (tools::build_info::is_release_build()) {
+        return;
+    }
+
+    // Applies only for Local Quorum consistency level.
```

Comment:
Why? Nothing below seems to depend on `cl`.

Reply:
removed the restriction by CL as agreed.